### ISSUE TITLE
Style link text

### ISSUE THREAD

**Issue by masakudamatsu** [masakudamatsu/mima#434]
related: #139 

## Problems
Currently, keyboard users cannot see whether link text is focused or not.

## Solutions
Use `styleFocusRing` and `removeFocusRing` in `cssUtilities.js` in the same way as `<ButtonCircle>` does

Or use `cssLinkText.js`
